src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBodyMapperTest.java

```diff
@@ -402,6 +402,26 @@ void shouldMapTemplateBodyWithOnlineResponse() {
             assertThat(requestBody).isEqualTo(expectedBody);
         }
 
+        @Test
+        void getDefendantNameFromClaimantIfMissingFromDefendantResponse() {
+
+            ccdCase.setRespondents(
+                ImmutableList.of(
+                    CCDCollectionElement.<CCDRespondent>builder()
+                        .value(SampleData.withRespondentPartyNameAsNull())
+                        .build()
+                ));
+
+            DocAssemblyTemplateBody requestBody = docAssemblyTemplateBodyMapper.from(
+                ccdCase,
+                userDetails
+            );
+
+            assertThat(ccdCase.getRespondents().get(0).getValue().getPartyName()).isEqualTo(null);
```

Comment:
You can simply to the following:

assertThat(ccdCase.getRespondents().get(0).getValue().getPartyName()).isNull();